draft-ietf-mls-protocol.md

```diff
@@ -2517,9 +2517,15 @@ The validity of a KeyPackage needs to be verified at a few stages:
 The client verifies the validity of a KeyPackage using the following steps:
 
 * Verify that the credential in the KeyPackage is valid according to the
-  authentication service and the client's local policy.  These actions MUST be
+  authentication service and the client's local policy. These actions MUST be
   the same regardless of at what point in the protocol the KeyPackage is being
-  verified.
+  verified with the following exception: If the KeyPackage is an update to
+  another KeyPackage, the authentication service MUST additionally validate that
+  the set of identities attested by the credential in the new KeyPackage is
+  acceptable relative to the identities attested by the old credential.
+
+* Verify that the set of identities attested by the credential is acceptable
+  to the application for the participant being updated.
```

Comment:
DIscussion on the virtual interim:

* This bullet seems redundant given the changes to the first bullet.  It is also out of place here in KeyPackage validation, since there might not be an update.
* TODO(@kkohbrok) Remove this bullet, then clear to merge

Reply:
Whoops, I think this wasn't supposed to be there in the first place. I'll remove it.